corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -3,6 +3,8 @@
 
 import itertools
 from casexml.apps.case.const import UNOWNED_EXTENSION_OWNER_ID
+# HACK: We need this import to initialize signal receivers. This should be handled in a different manner.
+import casexml.apps.case.signals  # noqa: F401
```

Comment:
I think one place this could go would be in a Django `AppConfig.ready` handler. Similar example

https://github.com/dimagi/commcare-hq/blob/30a8d7e6f8e2f2dddfde70d4c1979e6fe7cdb044/corehq/apps/es/app_config.py#L8-L10